### ISSUE TITLE
"light" version

### ISSUE THREAD

**Issue by CarlUpright** [GoldHEN/GoldHEN#97]
I would really love to have a "light" version, for a kid proof ps4. 

I mean, once loaded, the games would be unlocked, but no  goldhen menu, debug setting, nothing. 

I'd even like to hide internet settings, user guide and the web browser. 

In a perfect world, that would also hide the Friends, Events, Messages, Group, Profile and Settings menu. 

Is it something easy to do?

**Comment by Masamune3210** [comment]
Is modifying the entire top row of the Home Menu easy to do? I'm going to go with probably not lol

**Comment by BestDeveloper007** [comment]
I think the easiest thing is to get GoldHEN-Children, which turns on everything necessary in backround but does not show the Debug menu or any other new menu (just no new menu item, which is very easy to implement - about a minute of time)... But why hide everything else? Just password your profile and create a new account for your children. When the children have a new profile, it doesn't matter if they see friends (disabling the Internet is a good setting in the router where children cannot access)

**Comment by BestDeveloper007** [comment]
*Perhaps the easiest thing would be to teach children not to click on anything. :D

**Comment by CarlUpright** [comment]
> I think the easiest thing is to get GoldHEN-Children, which turns on everything necessary in backround but does not show the Debug menu or any other new menu (just no new menu item, which is very easy to implement - about a minute of time)... But why hide everything else? Just password your profile and create a new account for your children. When the children have a new profile, it doesn't matter if they see friends (disabling the Internet is a good setting in the router where children cannot access)

Where can i find this release?

**Comment by CarlUpright** [comment]
> *Perhaps the easiest thing would be to teach children not to click on anything. :D

Yeah, it would be easy if it was my childs, but i'm more looking into blocking random teens when i'm not there to watch.
